### PR TITLE
PSR-1174 Email content

### PR DESCRIPTION
**Pull request by kberec**

### CODE REVIEW

File: app/controllers/nonsipp/declaration/PsaDeclarationController.scala [hunk 1]
```diff
@@ -67,12 +68,14 @@ class PsaDeclarationController @Inject()(
       schemeDateService.schemeDate(srn) match {
         case None => Future.successful(Redirect(controllers.routes.JourneyRecoveryController.onPageLoad()))
         case Some(dates) =>
+          val now = schemeDateService.now()
           def emailFuture: Future[EmailStatus] =
```
Comment (cyeniceri) [inline app/controllers/nonsipp/declaration/PsaDeclarationController.scala:72]: Do we really need this "extra" emailFuture here ? It seems it's just calling sendEmail only.

File: app/controllers/nonsipp/declaration/PsaDeclarationController.scala [hunk 2]
```diff
@@ -86,25 +89,29 @@ class PsaDeclarationController @Inject()(
...
             Redirect(navigator.nextPage(PsaDeclarationPage(srn), NormalMode, request.userAnswers))
               .addingToSession((RETURN_PERIODS, schemeDateService.returnPeriodsAsJsonString(srn)))
-              .addingToSession((SUBMISSION_DATE, schemeDateService.submissionDateAsString(schemeDateService.now())))
+              .addingToSession((SUBMISSION_DATE, schemeDateService.submissionDateAsString(now)))
           }
       }
     }
 
-  private def sendEmail(name: String, email: String, taxYear: DateRange, schemeName: String)(
+  private def sendEmail(
```
Comment (cyeniceri) [inline app/controllers/nonsipp/declaration/PsaDeclarationController.scala:97]: same comment for the emailFuture as above

File: app/controllers/nonsipp/declaration/PsaDeclarationController.scala (hunk 1 above)
Comment (kberec) [reply app/controllers/nonsipp/declaration/PsaDeclarationController.scala:72]: I deliberately left it as it is in ER.

Comment (kberec) [reply app/controllers/nonsipp/declaration/PsaDeclarationController.scala:72]: We can refactor it later when everything works if you think it is necessary.

Comment (cyeniceri) [reply app/controllers/nonsipp/declaration/PsaDeclarationController.scala:72]: so shall we remove it if it's not required then ?

**Review by cyeniceri — APPROVED** [review]

File: app/controllers/nonsipp/declaration/PsaDeclarationController.scala (hunk 1 above)
Comment (cyeniceri) [reply app/controllers/nonsipp/declaration/PsaDeclarationController.scala:72]: I think they are unnecessary :) okay, we can remove them later.

**Pull request merged**